src/components/vetCenter/query.ts

```diff
@@ -137,9 +137,23 @@ export const formatter: QueryFormatter<VetCenterData, FormattedVetCenter> = ({
       featuredContentObject,
       ...vetCenterFeatureContentArray,
     ]
-    const formattedFeaturedContentArray = combinedArray.map((item) =>
-      queries.formatData(PARAGRAPH_RESOURCE_TYPES.FEATURED_CONTENT, item)
-    )
+    const formattedFeaturedContentArray = combinedArray.map((item) => {
+      const formattedItem = queries.formatData(
+        PARAGRAPH_RESOURCE_TYPES.FEATURED_CONTENT,
+        item
+      )
+      if (formattedItem?.link) {
+        const originalItem = item.field_cta || item
+        return {
+          ...formattedItem,
+          link: {
+            ...formattedItem.link,
+            url: originalItem.field_button_link?.url || formattedItem.link.url,
+          },
+        }
+      }
+      return formattedItem
+    })
```

Comment:
We can extract `withFallbackLink` to a helper file for use in here too, but I don't know how widespread its usage will actually be since I'm not really sure what the root cause here is. 😕 

Thoughts?

Reply:
I'm also wondering if there's any precedent for this in content-build.

Now that I'm looking at [the place where I had to deal with these "entity:" URIs a little while ago](https://github.com/department-of-veterans-affairs/next-build/blob/main/src/components/vamcSystemRegisterForCare/query.ts#L104), I think the answer to that is probably no. It probably didn't have the issue, because the commonality here is that all this data was fetched as centralized content using `entity_field_fetch`. My hypothesis is that it's `entity_field_fetch` that isn't properly translating the Drupal URIs to real URLs. In my instance, the only way I could fix it was by making a request to Drupal, because the data didn't exist anywhere else in the payload I got.